### DISCUSSION TITLE
Thank You to All Sponsors and How to Prioritize Your Issues!

### DISCUSSION THREAD

**Discussion by braden-w** [EpicenterHQ/epicenter#207] [Q&A]
Hi @DavidGP @cgbur @Wstnn @rkhrkh @doxgt @worldoptimizer!

I just wanted to thank you all for your sponsorship! It really means a lot to me. 🙏 I'm creating this thread as a place to formally say my thanks and gratitude.

Additionally, I want to have a way to prioritize the issues that matter most to you, my sponsors, in the next few ways.

## How to Help Me Prioritize Your Issues:

- Please create a new issue for any feature request or bug report you have.
- Once you have created the new issue, link it to this thread by adding a comment with the issue number, written in the format `#69` where the issue number is preceded by a #.
- I will prioritize the issues linked here higher in my to-do list.

Thank you once again for your support and contributions. Looking forward to your input and helping to make this project even better with your valuable feedback!

**Comment by Wstnn** [comment]
#145
Thanks bro. Awesome tool!

**Comment by worldoptimizer** [comment]
You are welcome. Let's get more people on board. When I use your interface, people always ask about it.

**Comment by rkhrkh** [comment]
In my opinion, you underestimate the groundbreaking significance of your application. To this day, many people invest heavily in reliable dictation programs and remain unaware of your revolutionary product. It is regrettable that this application is known only to insiders interested in artificial intelligence, rather than reaching a broader audience. 

Had I not persistently scoured AI on Reddit a year ago, I would still not have discovered your application. The ease it has brought to my workflow and the gratitude I feel are indescribable.

**Comment by DavidGP** [comment]
I totally agree. I use this wonderful application every day all day long in multiple languages. It is magic.

**Comment by braden-w** [comment]
Thank you guys so much for your kind words! :) Excited to develop and improve your workflows in the future!

**Comment by AlpSantoGlobalMomentumLLC** [comment]
Thanks, outstanding tool! I hope my sponsoring goes through ... this time. (I had a fight with github ;)

I'm curios, if that would be faster in the same quality ... As an option... https://console.groq.com/docs/speech-text

**Reply by braden-w** [reply]
Thank you so much @AlpSantoGlobalMomentumLLC ! It just came through :)

**Reply by AlpSantoGlobalMomentumLLC** [reply]
Thanks for giving me a feedback. I LOVE YOUR APP!!!

I just tested your App successfully with groq.com, as a proof of concept.

I'm not a programmer. That means I have no idea how I can change this for the Windows application.

Actually hard-coded it needs four lines. 
I tested my idea with the override in the browser and it worked. :D

OpenAI - Whisper Large **v2 = 36ct hour** - https://openai.com/blog/whisper/
groq         -  Whisper Large **v3 =  3ct hour** ~172x speed - https://console.groq.com/settings/billing

>         **if (!t.startsWith("gsk_"))**
>     ...
>                 **description: 'The API Key must start with "gsk_"',**
>        ....
>         **o.append("model", "whisper-large-v3"),**
>   ....
>             **try: ()=>fetch("https://api.groq.com/openai/v1/audio/transcriptions", {**
>

**Reply by braden-w** [reply]
Wow that's amazing! Thanks for making that progress—that will make it even faster for me to implement into the codebase! I'll take a look now, just recently finished addressing all the bugs that popped up after the recent notarization update!

**Reply by DavidGP** [reply]
So will we be able to choose the Groq API in Whispering?

Groq sounds very promising for speech recognition!

https://analyticsindiamag.com/ai-origins-evolution/groq-makes-llama-3-1-agi

**Reply by AlpSantoGlobalMomentumLLC** [reply]
> Wow that's amazing! Thanks for making that progress—that will make it even faster for me to implement into the codebase! I'll take a look now, just recently finished addressing all the bugs that popped up after the recent notarization update!

You are amazing! ;)

**Reply by braden-w** [reply]
Groq is now released in 4.8.0, hope this helps!! Check your settings :)

**Comment by vassovass** [comment]
Just thought to give props/kudos @braden-w for this, I use STT now a hell of a lot and sped up my workflow - especially with editing outputs.

For example last month I used it for 28k seconds which is just under 8 hours

![image](https://github.com/user-attachments/assets/e9e02d06-97ef-42bd-9e79-f71df9a892de)

**Reply by braden-w** [reply]
Woah that's awesome! So happy to hear that your workflow has been improved. Hopefully we've saved you several hours of typing!!

**Comment by rkhrkh** [comment]
#158

**Reply by braden-w** [reply]
Taking a look right now!

**Comment by rkhrkh** [comment]
#285

**Comment by rkhrkh** [comment]
#316

**Comment by rkhrkh** [comment]
#315

**Comment by braden-w** [comment]
Hey everyone, sorry for the delays in development. Have been busy with school once again and ironing out the next release using Tauri v2 in #301 . I hope to get a solid work session in later this week to wrap that up and work on the next set of features!!

**Comment by rkhrkh** [comment]
https://github.com/braden-w/whispering/issues/285
